compiler/src/test/java/com/squareup/anvil/compiler/dagger/AssistedFactoryGeneratorTest.kt

```diff
@@ -152,7 +152,8 @@ public final class AssistedServiceFactory_Impl implements AssistedServiceFactory
     }
   }
 
-  @Test fun `an implementation for a factory class is generated with intermixed assisted parameters`() {
+  @Test
```

Comment:
The formatter went rogue on random tests, didn't notice before 🙈 . Will revert these.

Reply:
Yep, keep it 👍🏻